lib/diagnostic.js

```diff
@@ -46,9 +46,7 @@ let scopes = [/* your response here */];
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
 //
-let fireOnAlderaan = function () {
-  // your code here
-};
+let fireOnAlderaan = deathStar.fire.bind(alderaan);
```

Comment:
close but you have to use .call instead of .bind. bind returns a bound function, but doesn't call it. call binds and calls the function